app/webpack/packs/application.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery'
+global.$ = $
+global.jQuery = $
+
+import 'bootstrap/dist/js/bootstrap';
+import '../src/javascript/sb-admin'
+
+import Rails from 'rails-ujs'
+Rails.start()
+global.Rails = Rails;
+
+import Turbolinks from 'turbolinks'
+Turbolinks.start()
+
```

Comment:
@marderer why did you rename `app/javascript` to `app/webpack`?

Reply:
@AleksSenkou because now I want using CSS with webpacker, not only javascript. I think that javascript is a bad name for the folder into which I will add styles and images, not only js.

Reply:
@marderer good notice. got it 👍